Dockerfile

```diff
@@ -1,21 +1,24 @@
-FROM alpine:edge
-
-RUN mkdir -p /etc/apk/repositories.d \
-  && echo http://dl-cdn.alpinelinux.org/alpine/edge/testing > /etc/apk/repositories.d/testing.list
+FROM alpine:3.7
```

Comment:
Why were we using edge before? We'd only do it if we needed something in particular from it... Just wondering if we're not breaking something by reverting it to a stable release.

Reply:
IIRC it was for the etcd-ctl package ...

Reply:
etcd is only in the testing repo and testing packages only gets published in edge not stable, because (by definition) testing packages are not stable!